ext/BeamTrackingBeamlinesExt/linear.jl

```diff
@@ -58,10 +58,7 @@ end
   # Sophia: this a thick corrector coil
   # In Fortran Bmad is g == 0 but dg != 0. 
   # In SciBmad this is g == 0 but K0 != 0.
-  gamma_0 = calc_gamma(bunch.species, bunch.Brho_ref)
-  K0 = get_thick_strength(bm1, L, bunch.Brho_ref)
-  mx, my, r56, d, t = LinearTracking.linear_dipole_matrices(K0, L, gamma_0; e1=bendparams.e1, e2=bendparams.e2)
-  return KernelCall(LinearTracking.linear_coast_uncoupled!, (mx, my, r56, d, t))
+  error("Undefined for tracking method $tm")
```

Comment:
```suggestion
  # Singularity for `g==0`
  error("Undefined for tracking method $tm")
```

We should add a comment as to why this isn't supported